Makefile

```diff
@@ -138,7 +145,7 @@ $(OUTPUT).elf	:	$(OFILES)
 
 #---------------------------------------------------------------------------------
 %.bin: %.elf
-	@$(OBJCOPY) -O binary $< $@
+	@$(OBJCOPY) --set-section-flags .bss=alloc,load,contents -O binary $< $@
```

Comment:
It does affect gateway, there's no second stage ROP for spider, so you need to use the one made specifically for CakeHax. e.g. http://dukesrg.no-ip.org/3ds/cakes?Decrypt9.dat

As for mset entry point, users have to use [CakesROP](https://github.com/mid-kid/CakesROP) to install our version of the first stage NVRAM exploit too (supports all regions and downgraded mset).

I'm not sure about bootstrap. @mid-kid can probably answer that.